### DISCUSSION TITLE
Use Whisper to convert Audio to text files and then use chatGPT to summarize

### DISCUSSION THREAD

**Discussion by pszgit** [chrislemke/ChatFred#85] [Ideas]
**Is your feature request related to a problem? Please describe.**
Problem  - convert audio to text using Whisper Open AI where we can see speaker and with an option to see time stamp or not.

**Describe the solution you'd like**
Generate text file or MS word file with transcribed text

**Describe alternatives you've considered**
MS teams transcribe and generation of VTT file

**Additional context**

**Comment by pszgit** [comment]
request for an update on this please

**Comment by neo-anderson** [comment]
It would be awesome to have this feature. Is it possible to get the bare minimum working? `Audio -> Transcription -> Summarize`. Time stamps and speakers are nice to have.
